Change: value8 => valueC (expecting revert on owner removal)

@package vaimo/composer-patches-target1

--- src/example.txt.org
+++ src/example.txt
@@ -5,7 +5,7 @@
 value5
 value6
 value7
-value8
+valueC
 value9
 value10
 value11
